editor/js/results.js

```diff
@@ -404,6 +416,28 @@ class ScorecardPresenter {
       self._updateCard(salesScorecard, salesMt, currentYear, salesSelected, hideVal);
       self._updateCard(equipmentScorecard, millionEqipment, currentYear,
         equipmentSelected, hideVal);
+
+      // Update dropdown values to match FilterSet
```

Comment:
For all of these if statements like `if (emissionsSubmetricDropdown && subMetric)`, do we need the null check? If the element is in `index.html`, let's assume it is there. We should check if `FilterSet` ever has it null. If it is null though, we should have it set to the default value as specified by `index.html`. I believe there are six such if statements.